### DISCUSSION TITLE
How to use this?

### DISCUSSION THREAD

**Discussion by pnvnd** [hughsie/python-cabarchive#12] [General]
Can you provide some examples in the README file to show how to use this package?  I can;t seem to figure out how to use this to extract or create CAB files.

For example, how do I create a CAB file with LZX compression like I would with Windows CABARC?
`cabarc.exe -m LZX:15 -o -i 4392 -s 8 n ./<cab_file_name>.cab <files>`

As well, how do I extract CAB files like I would with WIndow's MAKECAB?
`makecab /D CompressionType=LZX /D CompressionMemory=15 /D ReservePerCabinetSize=8 <file> <file>.cab`

For windows this is not an issue, but I would like to be able to do this on Linux and macOS as well.

**Comment by hughsie** [comment]
Do the docs help? https://python-cabarchive.readthedocs.io/en/latest/

**Comment by hughsie** [comment]
Also -- I'm not sure LZX is supported, I think only `NONE` and `MSZIP` are supported. If you need this I'd be happy to review a pull request adding support, but it's not something I need for the LVFS.
